src/components/common/icons/PlayIcon.js

```diff
@@ -0,0 +1,18 @@
+import React from "react";
+import PropTypes from "prop-types";
+
+import { Pressable } from "react-native";
+import Icon from "react-native-vector-icons/MaterialIcons";
+import { COLOR_PALETTE } from "../../../utils/constants";
+
+const PlayIcon = ({ onPress }) => (
```

Comment:
Are we going to re-use this everywhere we need a play button? If we're not, let's call this PlaylistPlayIcon, and have the onpress logic in here.